src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Home from './pages/Home';
+import Categories from './pages/Categories';
+import NotFound from './pages/NotFound';
+import NavBar from './componenets/NavBar/NavBar';
+import './App.css';
+
+function App() {
```

Comment:
- [ ] Kindly consider adding styles to the whole page so your design looks identical to the Zeplin design

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220923_180431](https://user-images.githubusercontent.com/25227022/192014559-0a5caf08-6994-47a6-91f9-f36b95835236.png)|![image](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png?raw=true)|